### DISCUSSION TITLE
softbody topology

### DISCUSSION THREAD

**Discussion by monkeygame** [NVIDIA-Omniverse/PhysX#63] [Q&A] [answered]
How do I modify the tetrahedron of softbody at run time?
Are there any related apis and precautions?

**Comment by preist-nvidia** [comment, marked as answer]
Hi @monkeygame - you will have to recook the softbody if you change the topology, see: https://nvidia-omniverse.github.io/PhysX/physx/5.1.2/docs/SoftBodies.html#cooking
